contributors/design-proposals/node/cpu-manager.md

```diff
@@ -360,6 +360,29 @@ func (p *dynamicPolicy) RemoveContainer(s State, containerID string) error {
    directly from the shared pool, is too simplistic.
     1. Mitigation: defer supporting this until a new policy tailored for
        use with `isolcpus` can be added.
+    2. A interim solution is to add a kubelet CLI command option to specify the 
+       explict cpu list reserved for the system, so the rest CPUs on the system
+       can be allocated exclusively for containers with Guranteed QoS. Since the 
+       CPUs to be used for containers becomes explict, the operator can simply 
+       define linux kernel boot parameter isolcpus using the same list. 
+       For example, if the operator wants to use cpu list 4-23 on the system
+       for containers with exclusive CPUs, add `isolcpuson=4-23` on the linux 
+       kernel boot parameter; add `--reserved-cpus=0-3` to the kubelet command.
```

Comment:
why to create new parameter instead of comply with cpuset specified in `--cgroup-root` ?
sysadmin might create kubepods root cgroup with cpuset.cpus which will not include 0-3.

Reply:
this is not about how to isolate the cpuset, using cgroup or isolcpu, it is about which cpuset  is isolated explicitly.

Reply:
@kad this option was primarily designed as interim workaround with --isolcpus kernel option to control the workload cpuset;  I would like to hear more on the cgroup approach,  https://github.com/kubernetes/enhancements/pull/1319 might be a better place to continue the discussion on cgroup approach.